proto/src/determined/api/v1/master.proto

```diff
@@ -25,6 +25,21 @@ message GetMasterResponse {
   string cluster_id = 3;
   // The cluster name.
   string cluster_name = 4;
+  // Telemetry status.
+  bool telemetry_enabled = 5;
+}
+
+// Get telemetry information.
+message GetTelemetryRequest {}
+// Response to GetTelemetryRequest.
+message GetTelemetryResponse {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "enabled" ] }
```

Comment:
can you check that this required field is working as intended?

Reply:
```
    "v1GetTelemetryResponse": {
      "type": "object",
      "properties": {
        "enabled": {
          "type": "boolean",
          "format": "boolean",
          "description": "Whether telemetry is enabled or not."
        },
        "segmentKey": {
          "type": "string",
          "description": "The key used for analytics in the UI."
        }
      },
      "description": "Response to GetTelemetryRequest.",
      "required": [
        "enabled"
      ],
      "title": "GetTelemetryResponse"
    },

```

Reply:
sweet.